content/news/posts/2025/reworking-the-update-assistant-module-v7-our-experience.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Reworking the \"Update Assistant\" Module V7: Our Experience"
+subtitle: "Transforming Complexity into Efficiency: The Story Behind Update Assistant V7"
+date:   2025-03-04
+authors: [ AlexisGuyomar ]
+icon: icon-rocket
+tags: [module revamp, update assistant, virtual scrolling, UX improvements, performance optimization, frontend, code quality, user experience]
```

Comment:
If I recall correctly, you have to use the tags available in tags.yml, could you confirm?